README.md

```diff
@@ -1,3 +1,70 @@
 # DCTNA Helm Chart
 
+<!-- This README.md is generated. -->
+
 This repository contains the Helm Chart for installing and configuring [DCTNA](https://github.com/philips-labs/dct-notary-admin).
+
+![Version: 0.1.0](https://img.shields.io/badge/Version-0.1.0-informational?style=flat-square) ![Type: application](https://img.shields.io/badge/Type-application-informational?style=flat-square) ![AppVersion: 0.1.0](https://img.shields.io/badge/AppVersion-0.1.0-informational?style=flat-square)
+
+A Helm chart for DCTNA.
+
+**Homepage:** <https://github.com/philips-labs/dct-notary-admin>
+
+## Maintainers
+
+| Name | Email | Url |
+| ---- | ------ | --- |
+| Marco Franssen | marco.franssen@philips.com |  |
+
+## Source Code
+
+* <https://github.com/philips-labs/dct-notary-admin>
+* <https://github.com/philips-labs/dctna-helm>
+
+## Requirements
+
+Kubernetes: `>=1.15`
```

Comment:
You missing here helm, and for test mac  linux including curl is required.

Reply:
This is generated by the tool. It is obvious a helm chart needs helm I assume.